runtime/compiler/ilgen/J9IlGeneratorMethodDetails.hpp

```diff
@@ -105,6 +105,7 @@ class OMR_EXTENSIBLE IlGeneratorMethodDetails : public OMR::IlGeneratorMethodDet
 
    virtual bool isOrdinaryMethod()     const { return true; }
    virtual bool isJitDumpMethod()      const { return false; }
+   virtual bool isJitDumpAOTMethod()   const { return false; }
```

Comment:
How does this work for ordinary methods? Given an `TR::IlGeneratorMethodDetails*` instance how does one determine if an ordinary method is an AOT method compilation?

Reply:
For ordinary methods, the decision to AOT or not happens in `preCompilationTasks`; the type of frontend that's created is used to determine whether the method is AOT'd or not. The reason I needed to do this is because of all the control logic; if a method is already in the SCC or it hasn't been downgraded, we won't do any AOT compile. So this is just to bypass all of that and force the compilation to be an AOT compile.